version.h

```diff
@@ -1 +1,4 @@
-#define HEXFIEND_VERSION 2.14.1
+#define HEXFIEND_VERSION_MAJOR 2
```

Comment:
I'd prefer not having another constant to maintain. you can just read the version from the CFBundleVersion key on the bundle's info.plist.

Reply:
Done.